README.md

```diff
@@ -1 +1,70 @@
-# Members-Only-Project
\ No newline at end of file
+<p align="center">
+  <a href="https://github.com/Fanger53/telegram_bot_Alfred">
+    <img src="https://res.cloudinary.com/growsurf-prod/image/upload/v1582211139/production/gnysw2objzekbagrqiax.png" alt="Logo" width="350" height="70">
+  </a>
+</p>
+ <h1 align="center">Members-Only-Project </h1>
+ 
+Basic project for Authentication, cookies, and user sessions. The main page is where it appears all the Posts and if the user signs in then the user should be able to see the authors for each post.
+
+# Built With
+-Ruby
+-Rails
+-Devise
+
+# Live Preview
```

Comment:
### [OPTIONAL]
- You can remove this from your readme since you do not have a demo link